07week/todolist/todolist/src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+  state = {
+  inputValue: '',
+  lists: []
+  }
+
+  inputChange(e){
+  this.setState({inputValue: e.target.value});
+
+  }
+  handleClick(){
+  if(this.state.inputValue){
+  const list = this.state.lists;
+  list.push(this.state.inputValue);
+  this.setState({lists: list});
+  this.setState({inputValue: ''})
+  }
+  }
+
+  deleteLastItem(){
+  const list = this.state.lists;
+  list.pop(this.state.lists)
+  this.setState({lists: list})
+  }
+
+
+  render() {
+  const createListItem = this.state.lists.map(function(value){
```

Comment:
this should be outside of the render() keep the render as pure as possible, so if you need to map through or do any 'logic' its best to keep it outside